java-checks/src/main/java/org/sonar/java/checks/CombineCatchCheck.java

```diff
@@ -47,23 +51,41 @@ public void visitNode(Tree tree) {
     for (CatchTree catchTree : ((TryStatementTree) tree).catches()) {
       for (CatchTree catchTreeToBeCompared : catches) {
         if (SyntacticEquivalence.areSemanticallyEquivalent(catchTree.block().body(), catchTreeToBeCompared.block().body())) {
-          reportIssue(catchTree, catchTreeToBeCompared);
+          reportIssueWithQuickFix(catchTree, catchTreeToBeCompared);
           break;
         }
       }
       catches.add(catchTree);
     }
   }
 
-  private void reportIssue(CatchTree catchTree, CatchTree catchTreeToBeCompared) {
-    String message = "Combine this catch with the one at line " + LineUtils.startLine(catchTreeToBeCompared.catchKeyword())
-      + ", which has the same body." + context.getJavaVersion().java7CompatibilityMessage();
+  private void reportIssueWithQuickFix(CatchTree catchTree, CatchTree catchTreeToBeCompared) {
+    String quickFixMessage = "Combine this catch with the one at line " + LineUtils.startLine(catchTreeToBeCompared.catchKeyword());
+    String issueMessage = quickFixMessage + ", which has the same body." + context.getJavaVersion().java7CompatibilityMessage();
```

Comment:
We should be careful about not having long messages since they have to appear in the contextual menu

Reply:
The `quickFixMessage` is the one going in the context menu, and is a portion of the original issue message, do you feel it could be too long? To me, we might have issues only with really big line numbers but I don't know if it makes sense to consider that case

Reply:
All good, it looks like the message is not too long
![Screenshot from 2023-01-30 14-26-02](https://user-images.githubusercontent.com/75226315/215490968-87509ac9-81b7-4b81-bb29-17784c9a1f2f.png)